diff --git a/BoardConfig.mk b/BoardConfig.mk
index 0ed4330..1054474 100644
--- a/BoardConfig.mk
+++ b/BoardConfig.mk
@@ -77,9 +77,11 @@ BOARD_USE_QCOM_PMEM := true
 BOARD_CAMERA_USE_GETBUFFERINFO := true
 BOARD_FIRST_CAMERA_FRONT_FACING := true
 BOARD_CAMERA_USE_ENCODEDATA := true
+BOARD_USES_HTC_CAMERA := true
+BOARD_USE_FROYO_LIBCAMERA := true
 
 BOARD_OVERLAY_FORMAT_YCbCr_420_SP := true
-USE_CAMERA_STUB := true
+USE_CAMERA_STUB := false
 
 # tenderloin- these kernel settings are temporary to complete build
 BOARD_KERNEL_CMDLINE := console=ttyHSL0,115200,n8 androidboot.hardware=qcom
